export function warn(msg) {
    try {
        console.warn('error:', msg);
    } catch(e) {}
}
